### ISSUE TITLE
Speaker Verification

### ISSUE THREAD

**Issue by abeguerra** [tyiannak/pyAudioAnalysis#8]
Should I be able to do speaker verification using aT.featureAndTrain & aT.fileClassification? I'm trying to train the SVM model using 7 wav files that I have for each of 5 users. Is that too small of a training set? I have another 3 wav files for each user that I'm using as a test set and its correctly classifying the test set. However, it's also classifying some of the wav files in this project as one of the users from my dataset (with a high probability, in one case ~.8).

Any advice/tips would be greatly appreciated.

**Comment by tyiannak** [comment]
Hi. In general, yes speaker verification is a classification task. However, using SVM with 7 training samples per class is too few and probably leads to overfitting.
